charts/seed-bootstrap/templates/gardener-seed-admission-controller/validatingwebhookconfiguration.yaml

```
@@ -0,0 +1,48 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: gardener-seed-admission-controller
+  labels:
+    app: gardener
+    role: seed-admission-controller
+webhooks:
+  - name: seed.admission.core.gardener.cloud
+    rules:
+    - apiGroups:
+      - apiextensions.k8s.io
+      apiVersions:
+      - v1beta1
+      - v1
+      operations:
+      - DELETE
+      resources:
+      - customresourcedefinitions
+    - apiGroups:
+      - extensions.gardener.cloud
+      apiVersions:
+      - v1alpha1
+      operations:
+      - DELETE
+      resources:
+      - backupbuckets
+      - backupentries
+      - containerruntimes
+      - controlplanes
+      - extensions
+      - infrastructures
+      - networks
+      - operatingsystemconfigs
+      - workers
+    failurePolicy: Fail
+    {{- if semverCompare ">= 1.15-0" .Capabilities.KubeVersion.GitVersion -}}
+    objectSelector:
+      matchLabels:
+        gardener.cloud/deletion-protected: "true"
+    {{- end -}}
```

This will render something like this:
```yaml
    failurePolicy: FailobjectSelector:
      matchLabels:
        gardener.cloud/deletion-protected: "true"namespaceSelector: {}
    clientConfig:
```


```suggestion
    {{- if semverCompare ">= 1.15-0" .Capabilities.KubeVersion.GitVersion }}
    objectSelector:
      matchLabels:
        gardener.cloud/deletion-protected: "true"
    {{- end }}
```